### ISSUE TITLE
Btn link box shadow

### ISSUE THREAD

**Issue by danijelGombac** [twbs/bootstrap#24711]
When shadow is enabled, btn-link change shadow on active state. Is this intended behavior?

https://codepen.io/danijelGombac/pen/KygLwE

**Comment by mdo** [comment]
This was fixed in #22998.

**Comment by danijelGombac** [comment]
I replaced .btn-link code from the PR, but the box shadow is still there.

https://codepen.io/danijelGombac/pen/KygLwE

When i added this, the box shadow is gone.

```
.btn-link-fix:not([disabled]):not(.disabled):active,
.btn-link-fix:not([disabled]):not(.disabled).active {
  box-shadow: none;
}
```
